tests/samples/bluetooth/samples_test_app/README.rst

```diff
@@ -0,0 +1,244 @@
+.. _samples_test_app:
+
+Bluetooth: BLE Master Test App
+##############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The BLE Master Test App is a Bluetooth Low Energy (BLE) central application designed for testing basic Bluetooth functionality in peripheral samples. This application acts as a tester with central role to validate BLE peripheral applications in the Nordic Connect SDK (NCS).
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+.. include:: /includes/tfm.txt
+
+The sample also requires a BLE peripheral device for testing. You can use any of the following peripheral samples:
+
+* :ref:`peripheral_uart`
+* :ref:`peripheral_hids_keyboard`
+* :ref:`peripheral_hids_mouse`
+* :ref:`peripheral_lbs`
+* :ref:`throughput`
```

Comment:
May be `ble_throughput` instead of `throughput`? This could fix the doc build error.